### DISCUSSION TITLE
control multiple mpd instances with multiple mympd instances

### DISCUSSION THREAD

**Discussion by payback007** [jcorporation/myMPD#314] [Q&A] [answered]
**Is your feature request related to a problem? Please describe.**
Is there a possibility to control multiple MPD instances with multiple myMPD instaces? 

**Describe the solution you'd like**
The idea is to have more than one player on several ports like
- mpd1 port 6600
- mpd2 port 6601
- mpd3 port 6602
- mpd4 port 6603

- control of mpd1 with mympd1 webserver port 80
- control of mpd2 with mympd1 webserver port 81
- control of mpd3 with mympd1 webserver port 82
- control of mpd4 with mympd1 webserver port 83

**Describe alternatives you've considered**
Up to now I tried to start two instances:
- mympd1.conf connected to port 6600 on webserver 80
- mympd2.conf connected to port 6601 on webserver 81

With native mpc client it is possible to control an individual port.

But the two instances of myMPD are influencing themselve. What I'm doing wrong? is it possible to connect with difference instances to different mpd ports or do I have to have for each myMPD instance one unique virtual machine for example?

**Comment by jcorporation** [comment, marked as answer]
Create for each myMPD instance a separate configuration file and add a individual varlibdir option. Look at https://github.com/jcorporation/myMPD/wiki/Configuration. Then copy the startup script and add these configuration as first parameter to the execute call, e.g. `ExecStart=/usr/bin/mympd /etc/mympd2.conf`.

By the way: on the roadmap is to support in version 6.7.0 with one myMPD instance multiple MPD instances and I think with MPD 0.22.0 you need also only one MPD instance and use partitions (also supported by myMPD 6.7.0 version).
`

**Comment by payback007** [comment]
Thank you very much, "varlibdir" was the missing information. It's working now as expected!

I also will see if 6.7.0 can fulfill my needs and test once it is available
